.github/PULL_REQUEST_TEMPLATE/make-it-live-pr.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Make it live PR
+about: Use this PR template for making changes from staging -> master
+title: 'Make it live'
+labels: make it live
+assignees: ''
+
+---
+
+## Changes Being Made Live
+
+<!-- What overall changes are being included in this "make it live" PR and are they all ready to be made live? -->
+
+## List of Commits/PR's Included
+
+<!-- List the staging commits/PR #'s that are included PR? with a general and quick summary of what each does -->
```

Comment:
```suggestion
<!-- List the staging commits/PR #'s that are included with a general and quick summary of what each does -- start by finishing the prompt below -->
```
To make it clear that below prompt would need to be modified to be relevant to the PR, what do you think?